web/ui/react-app/src/pages/alerts/AlertContents.tsx

```diff
@@ -0,0 +1,134 @@
+import React, { FC, useState, Fragment } from 'react';
+import { ButtonGroup, Button, Row, Badge } from 'reactstrap';
+import CollapsibleAlertPanel from './CollapsibleAlertPanel';
+import Checkbox from '../../Checkbox';
+import { isPresent } from '../../utils/func';
+
+export type RuleState = keyof RuleStatus<any>;
+
+export interface Rule {
+  alerts: Alert[];
+  annotations: Record<string, string>;
+  duration: number;
+  health: string;
+  labels: Record<string, string>;
+  name: string;
+  query: string;
+  state: RuleState;
+  type: string;
+}
+
+export interface RuleStatus<T> {
+  firing: T;
+  pending: T;
+  inactive: T;
+}
+
+export interface AlertsProps {
+  groups?: RuleGroup[];
+  statsCount: RuleStatus<number>;
+}
+
+interface Alert {
+  labels: Record<string, string>;
+  state: RuleState;
+  value: string;
+  annotations: Record<string, string>;
+  activeAt: string;
+}
+
+interface RuleGroup {
+  name: string;
+  file: string;
+  rules: Rule[];
+  interval: number;
+}
+
+const AlertsContent: FC<AlertsProps> = ({ groups = [], statsCount }) => {
+  const [state, setState] = useState<RuleStatus<boolean>>({
+    firing: true,
+    pending: true,
+    inactive: true,
+  });
+  const [showAnnotations, setShowAnnotations] = useState(false);
+
+  const toggle = (ruleState: RuleState) => () => {
+    setState({
+      ...state,
+      [ruleState]: !state[ruleState],
+    });
+  };
+
+  return (
+    <>
+      <ButtonGroup className="mb-3">
+        <Button active={state.inactive} onClick={toggle('inactive')} color="primary">
+          Inactive({statsCount.inactive})
+        </Button>
+        <Button active={state.pending} onClick={toggle('pending')} color="primary">
+          Pending({statsCount.pending})
+        </Button>
+        <Button active={state.firing} onClick={toggle('firing')} color="primary">
+          Firing({statsCount.firing})
+        </Button>
+      </ButtonGroup>
+      <Row className="mb-2">
+        <Checkbox
+          id="show_annotations"
```

Comment:
Let's call it `show-annotations` as that's closer to the naming style of the new UI.

Reply:
Yeah, ok. Not sure even is needed but will keep it in order to prevent future conflicts with other checkboxes on the page.

Reply:
Ah right, it's not even used - then remove it :)